core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -318,10 +318,15 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.isLearnAgainButton = function() {
+            var isConceptCard =
```

Comment:
Here and below: change booleans to something like stateIsShowingConceptCard, or conceptCardIsBeingShown, or similar, to distinguish them from the functions. This happens throughout most changes in https://github.com/oppia/oppia/pull/5265/files/86f735eb0b4a8d950127c29327d4cf267655fee7..b6c5785e243b5fcba310bdd2fe50f4b3f7344e7d